src/CadViewerManifold.tsx

```diff
@@ -136,7 +137,8 @@ const CadViewerManifold: React.FC<CadViewerManifoldProps> = ({
 }) => {
   const childrenCircuitJson = useConvertChildrenToCircuitJson(children)
   const circuitJson = useMemo(() => {
-    return circuitJsonProp ?? childrenCircuitJson
+    const rawCircuitJson = circuitJsonProp ?? childrenCircuitJson
+    return preprocessCircuitJsonWithFauxBoard(rawCircuitJson)
```

Comment:
bad name, it sounds like you're passing in circuit json with a faux board.

Just do `addFauxBoardIfNeeded(circuitJson)`